eng/pipelines/coreclr/runtime-nativeaot-outerloop.yml

```diff
@@ -162,6 +162,7 @@ extends:
         parameters:
           jobTemplate: /eng/pipelines/common/global-build-job.yml
           helixQueuesTemplate: /eng/pipelines/coreclr/templates/helix-queues-setup.yml
+          helixQueueGroup: cet
```

Comment:
Does this run both with and without CET, or will it run everything with CET only?

Reply:
Only the tests that run on windows_x64 would run with CET only, not with and without

Reply:
I was thinking it may be better to leave this job without CET (leave it unchanged) and add a new one that only tests `windows_x64` with CET and CFG enabled to make it more clear

Reply:
> Although it may be unnecessary to test with and without CET I'm assuming so I removed windows_x64 from this job and added it on a separate job on its own that enables CET and CFG

How do we test CoreCLR? Do we also only test with CET on Windows?

CFG produces different codegen so we might be losing codegen coverage if we only test with CFG and not without. No CFG is the mainstream scenario.

Reply:
My thoughts on this is that: 
- CET (as in "just CET, without EHCONT") is the default on Windows. As long as hardware/OS is new enough, this config will be tested
- no-CET will be tested on non-Windows10+ or non-x64 platforms. So far CET will be win-x64 only thing for quite a while.
- it is the CET+CFG (which also opts into EHCONT) that would not be normally tested, but still supported.
Thus we need some run/config to test this combination once in a while.

Reply:
> How do we test CoreCLR? Do we also only test with CET on Windows?

I believe we test it without CET mostly but we can optionally run the [runtime-cet](https://github.com/dotnet/runtime/blob/main/eng/pipelines/runtime-cet.yml) pipeline to test with CET. There's coverage for both with/without CET though. 

In that case I believe we can test CET and CET+CGF. Not sure if it's necessary to test without CET.

Reply:
Do we expect there could be bugs that repro with CET but don't repro with CET+CFG?

If not, my preference would be to test no-CET and CET+CFG, if for nothing else, to avoid duplicating the YAML.